addOns/graphql/src/main/resources/org/zaproxy/zap/extension/graphql/resources/Messages.properties

```diff
@@ -0,0 +1,19 @@
+# This file defines the default (English) variants of all of the internationalised messages
+
+graphql.api.action.importFile = Imports a GraphQL schema from a local file.
+graphql.api.action.importFile.param.file = The file that contains the GraphQL schema.
+graphql.api.action.importFile.param.target = The Target URL to override the server URL present in the schema.
+graphql.api.action.importUrl = Retrieves a GraphQL schema from an endpoint URL (via Introspection)
+graphql.api.action.importUrl.param.url = The URL locating the GraphQL endpoint.
+
+graphql.cmdline.file.help = Imports a GraphQL schema from the specified file name
+graphql.cmdline.url.help = Retrieves a GraphQL schema from the specified URL (via Introspection)
+graphql.cmdline.targeturl.help = The Target URL. Refer to the help for supported format.
+
+graphql.desc = Allows you to inspect and attack GraphQL endpoints
+graphql.topmenu.import.importgraphql = Import a GraphQL schema from the local file system
+graphql.topmenu.import.importgraphql.tooltip = The file must be a formally described GraphQL schema.
+graphql.topmenu.import.importremotegraphql = Import a GraphQL schema from an endpoint URL (via Introspection)
```

Comment:
This is an awfully long menu item entry (same with the one at L14).

Can they be shortened? It's fine for the tooltip to be more verbose for clarity.

May:

> Import GraphQL Schema from URL

(Still long, but much shorter)

Reply:
Sure. I was taking hints from the OpenAPI addon, hence the long menu entries. I will update the L16 entry, but keeping the L14 entry for uniformity (The corresponding OpenAPI menu entry is, "Import an OpenAPI definition from the local file system").

Reply:
Okay leave it as-is for now then. We can always tweak them all later. I didn't realize OpenAPI was like that.

Reply:
OH I guess you already changed it, all good! 😉